src/cmakeTools.ts

```diff
@@ -2023,8 +2005,8 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
 
         const buildOnLaunch = this.workspaceContext.config.buildBeforeRun;
         if (buildOnLaunch || isReconfigurationNeeded) {
-            const rc_build = await this.build([chosen.name]);
-            if (rc_build !== 0) {
+            const rcBuild = await this.build([chosen.name]);
```

Comment:
`rc` (result code) -> `result`.  More specifically, call this one `buildResult` to match the naming of the others.